src/cljs/harja/tiedot/urakka.cljs

```diff
@@ -373,3 +374,9 @@
   "Onko valitussa urakassa indeksi käytössä?"
   []
   (some? (:indeksi @nav/valittu-urakka)))
+
+(def urakan-tiedot-ladattu?
+  (reaction
+    (let [toimenpideinstanssit @urakan-toimenpideinstanssit
+          tehtavat @urakan-yksikkohintaiset-toimenpiteet-ja-tehtavat]
+      (boolean (and toimenpideinstanssit tehtavat)))))
```

Comment:
Monessa paikkaa tarvitaan urakasta ainakin nämä tiedot. Odottaisin näiden lataamista ennen kuin mitään urakan tietoja voidaan näyttää. Nyt on useammassa paikassa kaatuminen jos näitä esim. toimenpideinstanssit pitäisi renderöidä ennen kuin ne on ehditty ladata.